flake8_future_annotations/checker.py

```diff
@@ -86,9 +88,28 @@ def __init__(self, tree: ast.Module, filename: str) -> None:
     def run(self) -> Iterator[tuple[int, int, str, type]]:
         visitor = FutureAnnotationsVisitor()
         visitor.visit(self.tree)
-        if visitor.imports_future_annotations or not visitor.typing_imports:
+        if visitor.imports_future_annotations:
             return
 
-        imports = ", ".join(visitor.typing_imports)
-        lineno, char_offset = 1, 0
-        yield lineno, char_offset, ERROR_MESSAGE.format(imports), type(self)
+        lineno, char_offset, message = 1, 0, None
+
+        if visitor.typing_imports:
+            message = ERROR_MESSAGE_100.format(", ".join(visitor.typing_imports))
+        elif self.force_future_annotations:
+            message = ERROR_MESSAGE_101
+
+        if message is not None:
+            yield lineno, char_offset, message, type(self)
+
+    @staticmethod
+    def add_options(option_manager: Any) -> None:
```

Comment:
Can we use `OptionManager` instead of `Any`?
`from flake8.options.manager import OptionManager`

Reply:
I tried but mypy complains that there are no types exposed for flake8 module.

I found [this issue on the flake8 repo](https://github.com/PyCQA/flake8/issues/1560) which seems relevant.

If you prefer I can use `OptionManager` anyways and telling mypy to ignore it (making it typechecked as if it was `Any`).